app/src/main/java/com/ncnf/event/Event.java

```diff
@@ -1,31 +1,43 @@
 package com.ncnf.event;
 
+import android.content.Context;
+import android.widget.ImageView;
+
 import com.ncnf.organizer.Organizer;
+import com.ncnf.utilities.Location;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Date;
 import java.util.List;
 import java.util.UUID;
+import java.util.Date;
 
-public abstract class Event {
+public abstract class Event implements Comparable {
 
     public enum PubPriv {
```

Comment:
name could be more explicit, "Visibility" for example

Reply:
I agree

Reply:
Solved in Eloi's PR